### PR TITLE
shuffle docstring examples

### PR DESCRIPTION
**Pull request by RohitJain001**
#2887

### CODE REVIEW

**Comment by RohitJain001** [comment]
@abdrahmandiab Could you please review this PR once? I made changes only in the docstrings, I don't know why these three tests failed.
Thank you

**Review by abdrahmandiab — CHANGES_REQUESTED** [review]

File: ivy/container/random.py [hunk 1]
```diff
@@ -871,6 +871,36 @@ def static_shuffle(
         -------
         ret
             A container object, shuffled along the first dimension.
+
+        
+        Examples
+        --------
+
+        >>> x = ivy.Container(a=ivy.array([[9.8,7.6],[6.5,2.3]]), b=ivy.array([[0.9,2.4],[7.6,5.4]]))
```
[the remark this answers is not in the document]
Comment (RohitJain001) [reply ivy/container/random.py]: @abdrahmandiab I added a new commit. Please review it once

**Comment by RohitJain001** [comment]
@abdrahmandiab Could you please check why now these tests are failing, or Is it all correct?

**Comment by RohitJain001** [comment]
Hey @abdrahmandiab, Could you please review it?
I am trying to get it merged at my earliest and contribute more

**Review by abdrahmandiab — CHANGES_REQUESTED** [review]
Hi, @RohitJain001, sorry for the late reply, was a bit busy!

Please change the lines requested with this review and I'll proceed to check tests after. 
Thanks! 😄

**Comment by RohitJain001** [comment]
@abdrahmandiab Please review the latest commit changes once.

**Review by abdrahmandiab — CHANGES_REQUESTED** [review]
Hello @RohitJain001! 🙋🏻‍♂️

I've taken a more thorough look at your PR, and it seems to me as though there are a lot of points from the [Docstring Examples](https://lets-unify.ai/ivy/deep_dive/13_docstring_examples.html#docstring-examples) docs page that have not yet been implemented. 😅

It should also be noted that the examples in the [ivy/container/random.py](https://github.com/unifyai/ivy/blob/master/ivy/container/random.py) file that you have already edited might be reliant on the new examples you add in [ivy/functional/ivy/random.py](https://github.com/unifyai/ivy/blob/448855fe6fab444750b3e0850811b5d726f57c7b/ivy/functional/ivy/random.py#L474)


Specifically, please pay attention to:
* The docstring examples for [`shuffle`](https://github.com/unifyai/ivy/blob/448855fe6fab444750b3e0850811b5d726f57c7b/ivy/functional/ivy/random.py#L474) in **ivy/functional/ivy/random.py**.
* The docstring examples for [`shuffle`](https://github.com/unifyai/ivy/blob/448855fe6fab444750b3e0850811b5d726f57c7b/ivy/array/random.py#L338) in **ivy/array/random.py**.
* The docstring examples for [`shuffle`](https://github.com/unifyai/ivy/blob/448855fe6fab444750b3e0850811b5d726f57c7b/ivy/container/random.py#L1028) and [`static_shuffle`](https://github.com/unifyai/ivy/blob/448855fe6fab444750b3e0850811b5d726f57c7b/ivy/container/random.py#L981) in **ivy/container/random.py**.

Thank you for your hard work, looking forward to your updates! 🚀

**Comment by abdrahmandiab** [comment]
Hi @RohitJain001, 
Unfortunately I have to close this PR as it has been active for quite a long time!
Feel free to re-claim this function or any other one should you want to continue your application process at any time! 😅

**Pull request closed without merging**